src/main/java/org/folio/rest/impl/InvoiceLinesHelper.java

```diff
@@ -0,0 +1,37 @@
+package org.folio.rest.impl;
+
+import static org.folio.invoices.utils.HelperUtils.getInvoiceLineById;
+
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import me.escoffier.vertx.completablefuture.VertxCompletableFuture;
+
+import org.folio.rest.jaxrs.model.InvoiceLine;
+
+import io.vertx.core.Context;
+
+public class InvoiceLinesHelper extends AbstractHelper {
+
+  InvoiceLinesHelper(Map<String, String> okapiHeaders, Context ctx, String lang) {
+    super(getHttpClient(okapiHeaders), okapiHeaders, ctx, lang);
+  }
+
+  public CompletableFuture<InvoiceLine> getInvoiceLine(String id) {
+    CompletableFuture<InvoiceLine> future = new VertxCompletableFuture<>(ctx);
+    try {
+      getInvoiceLineById(id, lang, httpClient, ctx, okapiHeaders, logger)
+        .thenAccept(jsonInvoiceLines -> {
```

Comment:
The result is JsonObject representing only one invoice line. Please rename `jsonInvoiceLines` to something like `jsonInvoiceLine` and update log message below as well